### PR TITLE
Add moat integration

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add moat (BLE) integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/75643
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3549
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/moat.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ha_domain: moat
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [Moat](https://moat-tech.com/) devices into Home Assistant.
+
+## Supported devices
```
Comment (MartinHjelmare) [inline source/_integrations/moat.markdown]: I'd move the supported devices section below the config flow template.

File: source/_integrations/moat.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- [S2 Smart Temperature & Humidity Sensor](https://www.moat-tech.com/product/smart-climate-sensor/)
+
+The Moat integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
```
Comment (MartinHjelmare) [inline source/_integrations/moat.markdown:22]: I'd put this paragraph directly below the config flow template so it comes under the Configuration header.

File: source/_integrations/moat.markdown [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Moat
+description: Instructions on how to integrate Moat devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_zeroconf: true
```
Comment (MartinHjelmare) [inline source/_integrations/moat.markdown]: Should this be a front matter item for bluetooth discovery?

Comment (frenck) [reply source/_integrations/moat.markdown]:
```suggestion
ha_bluetooth: true
```

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks gents

**Pull request merged**